Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM alpine:3.15.0
-RUN apk --no-cache add ca-certificates git
+RUN apk --no-cache add ca-certificates git expat==2.4.4-r0
```

Comment:
I ran docker build with default Dockerfile and seems `2.4.4-r0` is already included in alpine:3.15.0
```
Step 1/5 : FROM alpine:3.15.0
 ---> c059bfaa849c
Step 2/5 : RUN apk --no-cache add ca-certificates git
 ---> Running in 3e2990ba22da
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/x86_64/APKINDEX.tar.gz
(1/7) Installing ca-certificates (20211220-r0)
(2/7) Installing brotli-libs (1.0.9-r5)
(3/7) Installing nghttp2-libs (1.46.0-r0)
(4/7) Installing libcurl (7.80.0-r0)
(5/7) Installing expat (2.4.4-r0)
(6/7) Installing pcre2 (10.39-r0)
(7/7) Installing git (2.34.1-r0)
Executing busybox-1.34.1-r3.trigger
Executing ca-certificates-20211220-r0.trigger
```
if it's fixed at os level I would avoid to hardcode dependencies

Reply:
I agree. This seems to just need a new build of the image to be made in order to patch the vulnerability. I will close this out.